### PR TITLE
Feat/Search Quick Result

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/components/Search/SearchQuickResult/index.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                  <UserDigest.Concise
+                    user={node}
+                    avatarSize="lg"
+                    nameStyle="tight"
+                  />
+                </Menu.Item>
+              </Fragment>
+            )
+        )}
+      {userEdges && userEdges.length > 0 && tagEdges && tagEdges.length > 0 && (
```
Comment (robertu7) [inline src/components/Search/SearchQuickResult/index.tsx:95]: Maybe we can add vars `hasUsers`, `hasTags` for the use of L95 and L73-L74.

File: src/components/Search/SearchQuickResult/index.tsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  if (
+    (!userEdges || userEdges.length <= 0) &&
+    (!tagEdges || tagEdges.length <= 0)
+  ) {
+    return null
+  }
+
+  return (
+    <Menu width={inPage ? undefined : 'md'}>
+      {!isSmallUp && (
```
Comment (robertu7) [inline src/components/Search/SearchQuickResult/index.tsx:67]: Can we reuse `inPage` here? Looks like these two components are only showing on mobile and `inPage` is always `true`.

**Review by robertu7 — APPROVED** [review]
Left some comments, overall looks good. 🚗

File: src/views/Search/index.tsx [hunk 3]
```diff
@@ -24,6 +25,7 @@ const Search = () => {
   const type = getQuery('type')
   const q = getQuery('q')
   const isSmallUp = useResponsive('sm-up')
+  console.log({ isSmallUp })
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/views/Search/index.tsx:28]: Will be removed next

File: src/components/Search/SearchQuickResult/gql.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import gql from 'graphql-tag'
+
+// TODO: Why this line don't work
+// import { TagDigest, UserDigest } from '~/components'
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/Search/SearchQuickResult/gql.ts:4]: it doesn't seem so

**Pull request merged**